mono/metadata/w32handle.h

```diff
@@ -117,7 +118,7 @@ gpointer
 mono_w32handle_new_fd (MonoW32HandleType type, int fd, gpointer handle_specific);
 
 gboolean
-mono_w32handle_close (gpointer handle);
+mono_w32handle_close (gpointer handle, MonoW32HandleType type);
```

Comment:
If you add the same argument to `mono_w32handle_ref`, you should add it to `mono_w32handle_duplicate` too.